pydicom/charset.py

```diff
@@ -54,12 +54,15 @@
     'GBK': 'GBK',  # from DICOM correction CP1234
 }
 
+# these encodings cannot be used with code extensions
```

Comment:
Can you just add a note saying these come from Table C.12-5 in part 3?

Reply:
Ah, I added a reference to part 5, but forgot to mention part 3. Can do this later.